src/popup/components/NotifyBell.vue

```diff
@@ -2,9 +2,9 @@
   <BtnIcon
```

Comment:
Now `BtnIcon` doesn't have a default slot, so the inner element won't be visible.

Reply:
Fixed.